tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.Faker/SqlScriptGenerator.cs

```diff
@@ -94,7 +98,7 @@ private static string GetValueAsString(object? value, Type propertyType)
             return TransformIntoSqlSafeString(value.ToString());
         }
 
-        if (propertyType == typeof(int) || propertyType == typeof(long))
+        if (propertyType == typeof(int) || propertyType == typeof(long) || propertyType == typeof(int?))
```

Comment:
The existing `typeof(int)` should already match `typeof(int?)`, is this (and the other nullable type checks) definitely needed?

Reply:
I don't know if its actually needed, I only know that when I didn't put this in it would error saying int? was an unsupported type